docs/pages/world/reference/system.mdx

```diff
@@ -0,0 +1,176 @@
+[//]: # "This file is autogenerated, do not change manually"
+
+## System
+
+[Git Source](https://github.com/latticexyz/mud/blob/main/packages/world/src/System.sol)
+
+**Inherits:**
+[WorldContextConsumer](/world/reference/world-context#worldcontextconsumer)
+
+_The System contract currently acts as an alias for `WorldContextConsumer`.
+This structure is chosen for potential extensions in the future, where default functionality might be added to the System._
+
+## SystemCall
```

Comment:
`SystemCall` is an internal util, should go into `/internals` since it's not recommended to be used directly in user code